src/UrqlClient.re

```diff
@@ -0,0 +1,380 @@
+type t;
```

Comment:
Just moved this file from `src/client/UrqlClient.re` to `src/UrqlClient.re`. Most other changes are minimal here.